charts/argo/templates/workflow-controller-deployment.yaml

```diff
@@ -43,6 +43,14 @@ spec:
           {{- if .Values.singleNamespace }}
           - "--namespaced"
           {{- end }}
+          {{- if .Values.controller.scaling.workflowWorkers }}
```

Comment:
You can use {{ with }} here to simplify things:

```
{{- with .Values.controller.podWorkers }}
- "--pod-workers"
- {{ . | quote }}
{{- end }}
```

Reply:
Yep definitely cleaner. Will implement.